manifests/k/KurtZimmermann/TweakPower/2.033/KurtZimmermann.TweakPower.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: KurtZimmermann.TweakPower
+PackageVersion: 2.033
+InstallerLocale: en-us
+InstallerType: exe
+InstallerSwitches:
+  Silent: /Si
+  SilentWithProgress: /Si
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://kurtzimmermann.com/files/winget/TweakPower2.033.exe
+  InstallerSha256: 0C27F58625D075DF3B2A6555942D679844E16009E19357F999195985870576A5
+ManifestType: installer
+ManifestVersion: 1.2.0
+ReleaseDate: 2023-02-18

```

Comment:
I don't see the ReleaseDate on the website, I may have missed it. Can you please share where you got this from?